prow/plank/controller.go

```diff
@@ -217,7 +218,7 @@ func (c *Controller) Sync() error {
 	}
 
 	var reportErrs []error
-	if c.ghc != nil {
+	if c.ghc != nil && reflect.ValueOf(c.ghc).IsNil() {
```

Comment:
this should be !, and also this should be unnecessary altogether if we fix it so we actually store `nil` -> `nil,nil` instead of `T*, nil`

Reply:
really? trying without it